nl_server/flask.py

```diff
@@ -57,16 +57,6 @@ def create_app():
     # Build the registry before creating the Flask app to make sure all resources
     # are loaded.
     reg = registry.build()
-
-    # Below is a safe check to ensure that the model and embedding is loaded.
-    server_config = reg.server_config()
```

Comment:
this is useful for production i think, do we want to condition this to be skipped only in integration-test?

Reply:
good idea. updated